src/Helper/FileHelper.php

```diff
@@ -18,6 +18,10 @@ class FileHelper
      */
     public static function getExtension($path)
     {
+        if ($path === null) {
```

Comment:
May need to change the `@param` to allow null instead of just string - I'd imagine the static analysis tools will moan about that once they're running.